get_facts_from_juniper.yml

```diff
@@ -0,0 +1,8 @@
+- name: Collect facts from Juniper device
```

Comment:
@ganeshhubale this looks good, although do we have any juniper devices to run this against?

Reply:
@kedark3 I have asked @ganeshhubale to upload this Playbook. I have plan to automate Ansible Network credentials.